.github/ISSUE_TEMPLATE/release.md

```diff
@@ -15,4 +15,5 @@ Like #4522, but for vX.X.X
 - [ ] Tag and release contrib vX.X.X
 - [ ] Prepare otelcol-releases vX.X.X
 - [ ] Release binaries and container images vX.X.X
-- [ ] Release the operator vX.X.X
\ No newline at end of file
+- [ ] Release the operator vX.X.X
```

Comment:
Perhaps we should state somewhere that the release manager does not need to release the operator/helm, but coordinate with the operator/helm maintainers to release it.

Reply:
Added some headers.  What do you think?